integrations/dd-sdk-android-compose/src/main/kotlin/com/datadog/android/compose/InteractionType.kt

```diff
@@ -4,9 +4,12 @@
  * Copyright 2016-Present Datadog, Inc.
  */
 @file:OptIn(ExperimentalMaterialApi::class)
+// suppressing deprecation for backward compatibility with SwipeableState
```

Comment:
seems like redundant comment, I don't see `@Suppress("Deprecation")` here

Reply:
yep, it's because it still in draft mode